prometheus-exporters/snmp-exporter/templates/alerts.yaml

```diff
@@ -17,8 +17,8 @@ spec:
 {{- end }}
 {{- end }}
 {{- $values := .Values -}}
-{{- if $values.snmp_exporter.pxdlrouternxos.alerts.enabled }}
-{{- range $path, $bytes := .Files.Glob "pxdl.alerts/*.alerts" }}
+{{- if $values.snmp_exporter.px_snmp.alerts.enabled }}
```

Comment:
This should be .px. and not px_snmp right? Otherwise the default has to be disabled. I think the variable should be named like the module name, otherwise it's hard to find out, where the variable is used.